ROADMAP.md

```diff
@@ -1,164 +1,113 @@
-reference implementation as well.
-
-Our work on in-toto implementations will continue to be in conjunction with the
-development of ITEs. Some exciting projects we expect to work on this year are:
-* supporting in implementations for the layouts that can verify attestations
-* adding ITE-9 reviewed predicates to one or more implementations
-* improving usability and simplifying the deployment of in-toto and TUF (ITE-2)
-  in software supply chains
-
-## Integrations
-
-Over the years, in-toto has been integrated into various other tools and
-systems. We plan to continue supporting their development and collaborating with
-other software supply chain security stakeholders. We are excited about the work
-we're doing with projects like Keylime, GUAC, and Sigstore, and we hope to talk
-about it more over the course of this year.
-
-Integrations we maintain ourselves will continue to get new features and
-support. Our Jenkins plugin recently received the capability to generate SLSA
-Provenance alongside its support for in-toto link metadata.
-
-## Community
-
-One of the efforts that has already started is a revamp of how the in-toto
-community is managed. A new starting point for all things in-toto was set up
-recently: https://github.com/in-toto/community. We believe this will help
-summarize the various in-toto projects and implementations in one place, as well
-as describe aspects of the community such as its governance, meeting cadence,
-contributing guidelines, and more.
-
-## Roadmap Review Schedule
-
-We have a new release schedule for our reviews as our roadmap now applies to the
-calendar year.
-
-- End of April 2023
-- End of August 2023
-- End of December 2023
-
-We will use these slots to release our roadmap reviews and depending on the
-development of various features, reviews may be accompanied by releases of our
-implementations.
\ No newline at end of file
+[Link to Roadmaps](/roadmap-reviews/)_
+
+## Ongoing Work
+
+Cross-collaboration with many external groups and projects, including:
+
+- OpenSSF and  groups
+  - SBOMit
+  - GitTUF
+  - Protobom
+- IETF
+  - SCITT
+- Package Repositories
+  - PyPi
+  - RubyGems
+
+## Recently Completed Work
+
+### Ecosystem
+
+Several major products and repositories now have support for in-toto attestations, including:
+
+- NPM
+- Homebrew
+- GitHub
+- GitLab
+
+### Specification and In-toto Enhancements (ITEs)
+
+- The in-toto v1.0 Specification was tagged June 5th 2023
+- ITEs to support layouts with the attestation framework were merged as `Draft`
+  - ITE-10: Supporting Contextual in-toto Attestations in Layouts
+  - ITE-11: Verifying Attributes in in-toto Attestations
+
+### Attestations
+
+- New Predicate Types
+  - Software Supply Chain Attribute Integrity (SCAI)
+  - Release
+  - Reference
+- Protobuf Language Bindings available
+  - Go
+  - Java
+  - Python
+
+### Witness
+
+- Donation of the project to in-toto during KubeCon '23 - Chicago
+- Improvement of CLO Monitor Scores to 99+
```

Comment:
https://clomonitor.io/projects/cncf/in-toto#witness (displays 100 ATM but 99+ is fine and could have been intentional :slightly_smiling_face:)
https://github.com/in-toto/witness/issues/334
https://github.com/in-toto/witness/pull/348

Fixing a false positive with CLO to be a true positive
https://github.com/in-toto/witness/issues/437
https://github.com/in-toto/witness/pull/451

Reply:
`go-witness` is 99. 😅